src/components/Datatable/DataRow.svelte

```diff
@@ -1,3 +1,23 @@
-<tr class="mdc-data-table__row">
+<script>
+import { createEventDispatcher } from 'svelte';
+
+const dispatch = createEventDispatcher()
+
+export let redirect = false
+
+const handleOnClick = event => {
+  dispatch('click', event)
+}
+</script>
+
+{#if redirect}
+  <style>
+    tr {
+      cursor: pointer;
+    }
+  </style>
+{/if}
+
+<tr on:click={handleOnClick} class="mdc-data-table__row">
```

Comment:
You can simplify this to...
```suggestion
<tr on:click class="mdc-data-table__row">
```
... if you simply want to pass up the click even to the parent component.